addons/web/static/src/search/filter_menu/custom_filter_item.xml

```diff
@@ -30,8 +30,7 @@
                         />
                     </select>
                     <span t-if="!('value' in selectedOperator)" class="o_generator_menu_value">
-                        <t t-if="false"/>
-                        <!-- <t t-if="fieldType === 'date'">
+                        <t t-if="fieldType === 'date'">
                             <DatePicker
                                 date="condition.value[0]"
                                 t-on-datetime-changed="onDateChanged(condition, 0)"
```

Comment:
onDateChanged does not exist

Reply:
onDateChang**e** exists but does not take any parameters

Reply:
I think you're looking at the wrong file:
- `custom_filter_item.js` has a `onDateChanged` method with 2 parameters
- `datepicker.js` has a `onDateChange` method with no parameters

Reply:
my bad